### ISSUE TITLE
SwiftLint 0.26 support

### ISSUE THREAD

**Issue by heidiproske** [ashfurrow/danger-ruby-swiftlint#103]
The latest `danger-ruby-swiftlint` [uses](https://github.com/ashfurrow/danger-ruby-swiftlint/blob/master/lib/version.rb#L5) `SwiftLint v0.25.1`.

A new `SwiftLint v0.26.0` is [now available](https://github.com/realm/SwiftLint/releases/tag/0.26.0). 

#### Motivation for the Update
I'm now using `Xcode 9.4` and have a PR that danger fails due to this bug https://github.com/realm/SwiftLint/issues/2231. Would like to get my PR merged so need to get this resolved and I'm hoping that with the latest SwiftLint it will go away.

I did try do a workaround by using `swiftlint:disable identifier_name` but then I get a local Xcode error saying `Superfluous Disable Command Violation: SwiftLint rule 'identifier_name' did not trigger a violation in the disabled region. Please remove the disable command. (superfluous_disable_command)` so I really need the latest SwiftLint :) 
 
#### What Next?
Based on previous PRs to this repo, it looks like it's usually just a 1 liner to bump the version as long as there are no breaking changes to the CLI from SwiftLint. [Based on the release notes](https://github.com/realm/SwiftLint/releases/tag/0.26.0) for `Swiftlint v0.26.0`, the only breaking change is that:
```
SwiftLint now requires Swift 4.0 or higher to build.
```
which shouldn't affect `danger-ruby-swiftlint` right?

I don't mind giving this a shot but not sure what to do. Do I just do something similar to how this PR bumped the version?
    https://github.com/ashfurrow/danger-ruby-swiftlint/pull/36/files

**Comment by heidiproske** [comment]
Aaaah thank you @ashfurrow ! :)

**Comment by heidiproske** [comment]
Aaaah thank you @ashfurrow !

 Btw, I'm using the swiftlint binary that ships with the danger-swiftlint gem (trying to move away from cocoapods) so I don't think I can just override the environment variable :)

**Comment by heidiproske** [comment]
I just tested this on several repos and the erroneous lint error bug for case enums is no longer happening, thank you @thii & @ashfurrow :)
